spec/Staking.md

```diff
@@ -351,6 +351,40 @@ share of "0xAB..CD" = Math.floor(35 * (1000 / (500 + 1000))) = 23
 remaing share (of author) = 110 - (11 + 23) = 76
 ```
 
+## ChangeParameters
+This transaction will change the common parameters when more than half of the stakeholders agree.
+It does not change the other fields of the scheme file, because there are fields related to the genesis block.
```

Comment:
It does not change other fields of the scheme file because there are fields related to the genesis block.